src/genie/libs/parser/iosxe/show_cts_sxp_connections_brief.py

```diff
@@ -0,0 +1,175 @@
+import re
+
+from genie.metaparser import MetaParser
+from genie.metaparser.util.schemaengine import Any, Optional
+
+
+# ===================================
+# Schema for:
+#  * 'show_cts_sxp_connections_brief'
+# ===================================
+class Show_Cts_Sxp_Connections_BriefSchema(MetaParser):
+    """Schema for show_cts_sxp_connections_brief."""
+
+    schema = {
+    "sxp_connections": {
+        "status": {
+            "sxp_status": str,
+            "highest_version": int,
+            "default_pw": str,
+            Optional("key_chain"): str,
+            Optional("key_chain_name"): str,
+            "source_ip": str,
+            "conn_retry": int,
+            "reconcile_secs": int,
+            "retry_timer": str,
+            "seq_export": str,
+            "seq_import":str
+        },
+        Optional("sxp_peers"): {
+            str: {
+                "source_ip": str,
+                "conn_status": str,
+                "duration": str
+            }
+        }
+    }
+    }
+
+
+# ===================================
+# Parser for:
+#  * 'show_cts_sxp_connections_brief'
+# ===================================
+class Show_Cts_Sxp_Connections_Brief(Show_Cts_Sxp_Connections_BriefSchema):
+    """Parser for show_cts_sxp_connections_brief"""
+
+    cli_command = ['show_cts_sxp_connections_brief']
```

Comment:
Its only one command no need to make a list

Reply:
This has been updated, but kept as a list to include 'show cts sxp connections vrf {vrf} brief'